src/robot/reporting/jsexecutionresult.py

```diff
@@ -28,19 +28,21 @@
 class JsExecutionResult(object):
 
     def __init__(self, suite, statistics, errors, strings, basemillis=None,
-                 split_results=None, min_level=None):
+                 split_results=None, min_level=None, auto_expand_ids=[]):
```

Comment:
Mutable default values are always a bit questionable. Would be better to use None or possibly () here.

Reply:
refactoring to None (deserialized to null in JS). I checked ECMAScript specification and also do some tests, which showed JavaScript handles [] exactly like null in for loops (None from Python).

Reply:
refactoring to None + implemented handling of None